server/resources/resource_management.go

```diff
@@ -480,7 +480,7 @@ func GetSingleResource(context middleware.Context, dataStore db.DB, resourceSche
 //GetSingleResourceInTransaction get resource in single transaction
 func GetSingleResourceInTransaction(context middleware.Context, resourceSchema *schema.Schema, resourceID string, tenantIDs []string, domainIDs []string) (err error) {
 	defer MeasureRequestTime(time.Now(), "get.single.in_tx", resourceSchema.ID)
-	var options *transaction.ViewOptions
+	options := &transaction.ViewOptions{Details: true}
 	r, ok := context["http_request"].(*http.Request)
 	if ok {
 		options = listOptionsFromQueryParameter(r.URL.Query())
```

Comment:
options are overwritten here

Reply:
intended. again, we don't want to change behavior of the API which allows overriting Options and for which join is the default behavior

Reply:
same as above